src/new/wallet.js

```diff
@@ -359,15 +358,11 @@ class HathorWallet extends EventEmitter {
         continue;
       }
 
+      // derive key to address index
       const derivedKey = key.deriveNonCompliantChild(addressIndex);
       const privateKey = derivedKey.privateKey;
 
-      // derive key to address index
-      const sig = crypto.ECDSA.sign(hash, privateKey, 'little').set({
-        nhashtype: crypto.Signature.SIGHASH_ALL
-      });
-
-      signatures[index] = sig.toString();
+      signatures[index] = tx.sign(privateKey).toString('hex');
```

Comment:
The old code here was doing `sig.toString()` before doing `sig.toDER()`. The new code is doing `sig.toDER().toString('hex')`, is it the same?

Reply:
bitcore-lib's [`Signature.toString()`](https://github.com/bitpay/bitcore/blob/master/packages/bitcore-lib/lib/crypto/signature.js#L189-L192) uses `toDER()` then `toString('hex')` internally.
This means we can just use `toString('hex')` here.